about.html

```diff
@@ -0,0 +1,156 @@
+        </div>
+        <div class="quicklink social">
+            <a href="#"><img src="assets/logos/Twitter.png" alt="Twitter Icon" /></a>
+        </div>
+        <div class="quicklink">
+            <p><a href="#"></a>English</p> 
+        </div>
+        <div class="quicklink">
+            <p><a href="#"></a>My Page</p> 
+        </div>
+        <div class="quicklink">
+            <p><a href="#"></a>Logout</p> 
+        </div>
+    </div>
+    <!-- Navbar section -->
+     <div class="humberger">
+        <div class="line1"></div>
+        <div class="line2"></div>
+        <div class="line3"></div>
+     </div>
+     <div id="navbar" class="navbar">
+        <div class="logo">
+            <a href="index.html"><img src="assets/logos/Logo.png" alt="page logo" /></a>
+        </div>
+        <ul>
+            <li class="links home-page"><a href="index.html">Home</a></li>
+            <li class="links"><a href="about.html">About</a></li>
+            <li class="links"><a href="#">Program</a></li>
+            <li class="links"><a href="#">Join</a></li>
+            <li class="links"><a href="#">Sponsor</a></li>
+            <li class="links last-link"><a href="#">News</a></li>
+            <li class="links focused"><a href="#">CC Campaign</a></li>
+        </ul>
+     </div>
+</header> 
+<main>
+    <section id="about">
+      <p class="quick-link">
+        <a class="top-link" href="index.html">Home</a><span> &#62; </span>
+          <a class="top-link" href="about.html">About</a>
+      </p>
+      <div class="about-me">
+        <h2>"hello! Sharing world"</h2>
+          <h1>Creative commons global summit 2022</h1>
+          <p class="bio">
+            the global summit brings together thecommunity of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country everyt two years. <br />
+            This year, we hope to expand our invitation list including
+            organizations and individuals who want to work with us on shared
+            projects that advance the cause of the commons, free culture and
+            open knowledge. <br />
+            So if you're active and engaged in the worlds of open content and
+            knowledge -- free software advocates, Wikipedians, Open Knowledge,
+            galleries, libraries, museums, archives, governements and
+            foundations, lawyers, and activists -- we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together.
+          </p>
+          <p class="ad-det">
+            Please contact us per email for any further question <br />
+            about CC global Summit 2015!
+          </p>
+          <p class="about-email">ccsummit2015@cckorea.org</p>
+      </div>
+      <!-- Creative Commone Section -->
+      <section id="co-logo">
+        <h3>Creative Commons Global summit 2022 Logo</h3>
+        <hr />
+        <p class="mobile">
```

Comment:
This part looks good, but you can update the background color and sizing. So please make sure to follow the original design idea.

![image](https://user-images.githubusercontent.com/34779928/196794423-25306b94-e713-4e47-b586-9d574db31d3e.png)